src/components/app/ConfigEditor.vue

```diff
@@ -8,6 +8,7 @@
         <v-col class="py-0">
           <v-text-field
             v-model="globalConfig.numDaysBack"
+            class="input-element"
```

Comment:
Not sure I'm remembering this

Reply:
OK, `min-width` to not getting this:

![image](https://user-images.githubusercontent.com/17686879/98670480-72cd8600-235b-11eb-8ca0-5a4c8a1e1e45.png)